sqlmesh/core/node.py

```diff
@@ -33,17 +36,19 @@ class IntervalUnit(str, Enum):
     MONTH = "month"
     DAY = "day"
     HOUR = "hour"
+    HALF_HOUR = "half_hour"
+    QUARTER_HOUR = "quarter_hour"
     MINUTE = "minute"
 
     @classmethod
-    def from_cron(klass, cron: str, sample_size: int = 10) -> IntervalUnit:
+    def from_cron(klass, cron: str, sample_size: int = DEFAULT_SAMPLE_SIZE) -> IntervalUnit:
         croniter = CroniterCache(cron)
         samples = [croniter.get_next() for _ in range(sample_size)]
         min_interval = min(b - a for a, b in zip(samples, samples[1:]))
         for unit, seconds in sorted(INTERVAL_SECONDS.items(), key=lambda x: x[1], reverse=True):
-            if seconds <= min_interval:
+            if seconds <= min_interval.total_seconds():
                 return unit
-        raise ConfigError(f"Invalid cron '{cron}': must have a cadence of 1 minute or more.")
+        raise ConfigError(f"Invalid cron '{cron}': must have a cadence of 5 minutes or more.")
```

Comment:
Do we prohibit using `minute` grain anywhere?

Reply:
yea, it's not an enum anymore